OpenRA.Game/Activities/CallFunc.cs

```diff
@@ -19,22 +19,15 @@ public class CallFunc : Activity
 		public CallFunc(Action a, bool interruptable)
 		{
 			this.a = a;
-			this.interruptable = interruptable;
+			IsUninterruptible = interruptable;
```

Comment:
Shouldn't this be `!interruptable`?
Which brings me to this: Why `IsUninterruptbile` instead of `IsInterruptible`?